lib/mix/tasks/scenic.gen.scene.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Mix.Tasks.Scenic.Gen.Scene do
+  @shortdoc "Creates scaffolding for a new Scenic scene."
+
+  @moduledoc """
+  Generates Scene boilerplate for a new scene.
+
+  ```bash
+  mix scenic.gen.scene scene_name
```

Comment:
Update documentation
```suggestion
  mix scenic.gen.scene SceneName
```